### ISSUE TITLE
Consider "_preds" suffix

### ISSUE THREAD

**Issue by KevinMusgrave** [KevinMusgrave/pytorch-adapt#11]
Anytime softmax is applied to "_logits", add "_preds".
